lib/rspec/core/did_you_mean/suggestions.rb

```diff
@@ -0,0 +1,55 @@
+module RSpec
+  module Core
+    module DidYouMean
+      # Service object to provide did_you_mean suggestions
+      # based on https://github.com/yuki24/did_you_mean
+      class Suggestions
+        CUT_OFF = 0.834 # Lowest acceptable proximity to be considered probable
+        MAX_SUGGESTIONS = 3 # Maximum number of suggestions that can be provided.
+        attr_reader :relative_file_name, :exception
+
+        def initialize(relative_file_name, exception)
+          @relative_file_name = relative_file_name
+          @exception = exception
+        end
+
+        # provide probable suggestions if a LoadError
+        def call
+          return unless RUBY_VERSION.to_f >= 2.0
+          return unless exception.class == LoadError
+
+          probables = find_probables
+          return unless probables.any?
+
+          short_list = probables.sort_by { |_, proximity| proximity }.reverse[0...MAX_SUGGESTIONS]
+          formats short_list
+        end
+
+        private
+
+        def formats(short_list)
+          rspec_format = short_list.map { |s, _| "rspec ./#{s}" }
```

Comment:
I think we have some logic for this somewhere

Reply:
I have not been able to find it.  Is your concern that sometimes the correct prefix is not 'rspec ./'?